packages/tpetra/core/src/Tpetra_FECrsGraph_def.hpp

```diff
@@ -42,11 +42,13 @@
 #ifndef TPETRA_FECRSGRAPH_DEF_HPP
 #define TPETRA_FECRSGRAPH_DEF_HPP
 
-#include <type_traits>
-#include "Tpetra_CrsGraph.hpp"
+#include "Tpetra_FECrsGraph.hpp"
 #include "Tpetra_Details_Behavior.hpp"
 #include "Tpetra_Details_getEntryOnHost.hpp"
+#include "Tpetra_Details_makeColMap.hpp"
```

Comment:
Is this include needed?  It appears the code that calls it is commented out below.

Reply:
Ah, good catch. I went through a few impl cycles, where at some point I tried to build a col map, but then backed off. This was supposed to be removed.